pom.xml

```diff
@@ -42,7 +42,7 @@
     <guava.version>18.0</guava.version>
     <slf4j.version>1.7.7</slf4j.version>
     <!-- plugin versions -->
-    <checkstyle.plugin.version>2.14</checkstyle.plugin.version>
+    <checkstyle.plugin.version>2.15</checkstyle.plugin.version>
```

Comment:
If we are going to call out a new version, maybe (putting on my scary rubber @awoods mask) there's a separate ticket here to note that. If we're removing the version, it seems fine as part of this.